docs/Employer/5-Employment.md

```diff
@@ -108,3 +109,4 @@ Name | Type | Description
 `income.unit` | `string` | The income unit of payment. Options: `yearly`, `monthly`, `semi_monthly`, `bi_weekly`, `weekly`, `daily`, `hourly`, and `fixed`.
 `income.amount` | `integer` | The income amount in cents.
 `income.currency` | `string` | The currency code.
+`income.effective_date` | `string` | The effective date of the individual's income.
```

Comment:
Nit: not sure if this is super clear. Is there some copy to more clearly indicate it's the date this income amount went into effect? 🤷🏾

Reply:
How about: "The date the `income` object went into effect"

Reply:
The date the income amount went into effect would make sense to me!